lib/datadog/core/git/ext.rb

```diff
@@ -0,0 +1,34 @@
+# typed: true
+module Datadog
+  module Core
+    module Git
+      # Defines constants for Git tags
+      module Ext
```

Comment:
I think this file should belong to the `CI` product, not core.

Reply:
Any thoughts @juan-fernandez ? Does this have legitimate use outside CI, or is it very CI specific?